deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -23,17 +23,32 @@ export HTTPS_PROXY=${HTTPS_PROXY:=""}
 export NO_PROXY=${NO_PROXY:=""}
 
 function remaining_fields() {
-    local RESPONSE="$(curl -XGET -s \
+    local RESPONSE
+    readonly RESPONSE="$(curl -XGET -s \
         -u "${SUMOLOGIC_ACCESSID}:${SUMOLOGIC_ACCESSKEY}" \
         "${SUMOLOGIC_BASE_URL}"v1/fields/quota)"
 
-    echo "${RESPONSE}" | jq '.remaining'
+    echo "${RESPONSE}"
 }
 
 # Check if we'd have at least 10 fields remaining after additional fields
 # would be created for the collection
 function should_create_fields() {
-    local REMAINING=$(remaining_fields)
+    local RESPONSE
+    readonly RESPONSE=$(get_remaining_fields)
```

Comment:
The function name is wrong, you probably forgot to change it there in the definition?

Reply:
Good catch! Changed